buildout.cfg

```diff
@@ -37,7 +37,7 @@ primary_indexer_queue_type = Simple
 primary_indexer_queue_server = true
 primary_indexer_queue_worker = true
 primary_indexer_queue_worker_processes = 16
-primary_indexer_queue_worker_chunk_size = 512
+primary_indexer_queue_worker_chunk_size = 265
```

Comment:
not sure it matters, but should not that be 256

Reply:
it doesn't matter, but we have to revert this before merge.